tests/sentry/grouping/snapshots/test_variants/test_event_hash_variant/newstyle:2019_04_17/javascript_exception_fallback_to_message.pysnap

```diff
@@ -0,0 +1,24 @@
+---
+created: '2019-04-17T20:26:40.111823Z'
+creator: sentry
+source: tests/sentry/grouping/test_variants.py
+---
+app:
+  hash: None
+  component:
+    app (exception of system takes precedence)
+      exception (ignored because hash matches system variant)
+        type*
+          u'Error'
+        value* (stripped common values)
+          u'Loading chunk <int> failed.\n(timeout: <url>'
```

Comment:
This looks a bit strange as it strips the parenthesis, but it's unlikely for this to ever produce false positives.